app/services/teacher_training_public_api/sync_courses.rb

```diff
@@ -45,10 +45,7 @@ def create_or_update_course(course_from_api, recruitment_cycle_year, incremental
       @updates.merge!(courses: true) if !incremental_sync && course.changed?
 
       course.save!
-
-      if new_course
-        SetOpenOnApplyForNewCourse.new(course).call
-      end
+      course.open! if new_course
```

Comment:
We retain the `new_course` check because we don't want to accidentally open old courses or courses which for some reason have been closed on Apply (we'll revisit that when we rip out `open_on_apply` altogether)